### DISCUSSION TITLE
Enhancement: Add additional self hosting options

### DISCUSSION THREAD

**Discussion by cosark** [danny-avila/LibreChat#3680] [Feature Requests & Suggestions]
### What features would you like to see added?

https://github.com/danny-avila/LibreChat/pull/3605

### More details

Currently there are only 3 very expensive hosting options on the readme

**Comment by danny-avila** [comment]
I appreciate you bringing this up but I have to decline adding more options.

They unfortunately lead to many questions of these hosting services, and I can only vouch for digitalocean, whose instructions can be applied to any other linux-based, non-abstracted VPS, for example, hetzner cloud, which is considerably cheaper than digitalocean but less feature-rich (meaning more "know-how" is a requirement).

Due to this, I can't help users who choose to deploy on these options that I don't use. If you are the maintainer of the repocloud image, and are open to help users troubleshoot those deployment, I'd be more inclined to include it, saying nothing of the referral codes that seem to benefit the creator of the link.

**Comment by cosark** [comment]
The image is maintained by myself and also the RepoCloud team. They have dozens of LibreChat users already whose issues are quickly resolved. They can definitely provide support to any users who have any issues. And there's no affiliate code attached to that link, it's just the raw app_id direct link. I'd encourage you to try it for yourself if you have any doubt. It's a great service and much easier than a VPS deployment, for half the cost of Digital Ocean.
